.github/workflows/flatpak.yaml

```diff
@@ -0,0 +1,23 @@
+on: tag
+
+jobs: 
+  build: flatpack
+  - uses: actions/checkout@v2   # automatically clone the repository
+    runs-on: ubuntu-20.04  # verified on ubuntu-20.04 and should run on any debian distro
+    
+    - name: install Flatpak
+      run: >
+        sudo apt install flatpak gnome-software-plugin-flatpak && flatpak remote-add --if-not-exists flathub https://flathub.org/repo/flathub.flatpakrepo && flatpak install flathub org.gnome.Platform//3.36 org.gnome.Sdk//3.36 org.flatpak.Builder
+      
+    - name: build Flatpak  # run command in an empty directory with org.flatpak.Gourmet.yml placed in it
+      run: >
+        flatpak-builder build-dir org.flatpak.Gourmet.yml 
+        
+    - name: run Flatpak  # run in same directory as buildFlatpak
+      run: >
+        flatpak-builder --run --socket=fallback-x11 build-dir org.flatpak.Gourmet.yml gourmet
+     
+# Until issue # 107 is fixed, in the directory Gourmet was built in, open the settings.py in the following path
```

Comment:
When this is merged, could you add a comment to #107 to say that this file should be updated to remove these comments?

Reply:
ok, that is done now, thank you